README.md

```diff
@@ -45,27 +48,33 @@ Fuzz tests for pretty-printing.
 Index-usage assertion tests. These tests specify queries and expected index utilization.
 Requires a running mongod.
 
-`cargo test --features=index,generated --package=e2e-tests -- --test-threads=1` from the main directory
```

Comment:
We no longer require single-threaded testing since all tests are now thread-safe 🎉